content/get_start/UseCase_Workflows/revit.md

```diff
@@ -0,0 +1,108 @@
+{{% /notice %}}
+
+- Revit Systems Analysis OpenStudio model 
+    - Offers a streamlined workflow 
+    - Effectively translates most of the gbXML content into BEM using OpenStudio measures. 
+    - Various export modes available like a) Use Conceptual Masses and Building Elements (can be preferred for SD models), b) Use Rooms or Spaces (can be preferred for DD and CD models)
+
+{{% notice info %}}
+"Use Rooms or Spaces" allows the BEM model to inherit all rooms and incorporate the pre-assigned thermal zoning from the BIM model. In the BIM model multiple spaces can be associated with the same thermal zone indicating that they are controlled by a single thermostat and this is reflected in the BEM model.
+{{% /notice %}}
+
+
+ ##### **4. Apply 90.1 PRM method**
+Apply the 90.1 PRM meausre in OpenStudio by following the steps provided in the initial sections of this user guide based on the method adopted. 
+
+Key information required to run the PRM measure include space types for lighting power density, building area type for window to wall ratio, HVAC system selection and water heater system, and process loads. 
+Thus, before running the measure, it is important to check the exported BEM model for any missing information and addressing them by following the steps provided for Model Preparation below to ensure a successful measure run. 
+
+ - Model Preparation 
+     - Standard Space Type Mapping 
+       
+       Mapping of the space types is a common challenge across different export routes and design phases.   
+       - Cross validate different building area types. In PRM there are three building type categories used to determine: the baseline window-to-wall-ratio (WWR), HVAC system type, and service hot water (SHW) system type that needs to have the same type assigned to them. For example, if the section of a building model is categorized as an "Office (<=5000 ft^2)", it must not be classified as a "Multifamily" building type to determine the baseline model SHW system.
+       - If required, manually map space types from Revit to those listed in PRM to align them accurately. 
+
+     - HVAC System Integration 
+       
+       One of the key data elements often missing in the initial stage of design is related to the HVAC systems. For example, ideal load air system commonly defined for SD phase since a complete HVAC system is not typically necessary. Additionally, various export methodologies might not be able to translate the HVAC system design into energy models. 
+
+       - Address missing HVAC system data including details like fan power distribution ratios, crucial for a fully automated solution. 
+       - A solution that can be adopted is generating the export using Revit Systems Analysis's built-in functionality. This feature generates a complete OpenStudio model using a gbXML based schema that includes a comprehensive HVAC system description, making it ready for the OSSTD-PRM generation process. 
+
```

Comment:
@AmmarDeh if it is not too much work, could you provide some screen shot on Revit System Analysis HVAC assignment?